lumen/ai/agents.py

```diff
@@ -1000,7 +1000,7 @@ async def _extract_spec(self, spec: dict[str, Any]):
             vega_spec["width"] = "container"
         if "height" not in vega_spec:
             vega_spec["height"] = "container"
-        return {'spec': vega_spec, "sizing_mode": "stretch_both", "min_height": 500}
+        return {'spec': vega_spec, "sizing_mode": "stretch_both", "min_height": 300}
```

Comment:
Made it less since overflow starts clipping. It stretches if there's available space.